plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -713,7 +713,7 @@ public JdbcOutputTableHandle beginInsertTable(ConnectorSession session, JdbcTabl
                     columnTypes.build(),
                     Optional.of(jdbcColumnTypes.build()),
                     Optional.of(remoteTemporaryTableName),
-                    pageSinkIdColumn.map(ColumnMetadata::getName));
+                    pageSinkIdColumn.map(column -> identifierMapping.toRemoteColumnName(connection, column.getName())));
```

Comment:
Is it possible to test it?

Reply:
Sorry, missed your comment. It is tested in another repo (plugin), but I can take a look, maybe I could add a short one here.